pyproject.toml

```diff
@@ -9,20 +9,24 @@ authors = [
     { name="Thomas Uebensee", email="Thomas Uebensee <ext.thomas.uebensee@exasol.com>" }
 ]
 
+[project.scripts]
+exaslc-ci = 'exasol.slc_ci.cli.main:main'
+
 [tool.poetry]
 requires-poetry = ">=2.1.0"
 packages = [
     {include = "README.md"},
     {include = "LICENSE"},
     {include = "exasol_script_languages_container_ci"},
+    {include = "exasol"},
 ]
 
 [tool.poetry.dependencies]
 python = ">=3.10.0,<4.0"
 click = "^8.1.7"
 GitPython = ">=3.1.0"
 exasol-script-languages-container-tool = "^3.0.0"
-exasol-integration-test-docker-environment = "^4.0.1"
+exasol-integration-test-docker-environment = "4.0.1"
```

Comment:
Why did you drop the ^

Reply:
There is a problem with ITDE 4.1.0 and exaslct. We discussed on the phone this week.